code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/Message.java

```diff
@@ -24,14 +20,19 @@
 
 import com.adobe.marketing.mobile.AdobeCallback;
 import com.adobe.marketing.mobile.launch.rulesengine.RuleConsequence;
-import com.adobe.marketing.mobile.messaging.MessagingConstants.EventDataKeys.MobileParametersKeys;
+import com.adobe.marketing.mobile.messaging.internal.MessageRequiredFieldMissingException;
```

Comment:
https://github.com/adobe/aepsdk-sample-app-android/blob/main/Sample-App/app/src/main/java/com/adobe/marketing/mobile/sampleapp/MainApp.java#L15

In 1.x Messaging APIs are exposed inside `com.adobe.marketing.mobile` package.

Reply:
Is this a new API we expose in 2.0.0?